prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -116,6 +142,19 @@ class HttpUtilities {
             return """{"error": "$message"}"""
         }
 
+        private fun addHeaderIfModified(
```

Comment:
Slightly subtle point. The settings table is initially empty and there isn't a last modified time. So, this code only adds a lastModified header if a last modified time exists.